openwisp_monitoring/db/backends/influxdb/client.py

```diff
@@ -117,13 +118,16 @@ def write(self, name, values, **kwargs):
         if isinstance(timestamp, datetime):
             timestamp = timestamp.isoformat(sep='T', timespec='microseconds')
         point['time'] = timestamp
-        self.get_db.write(
-            {'points': [point]},
-            {
-                'db': kwargs.get('database') or self.db_name,
-                'rp': kwargs.get('retention_policy'),
-            },
-        )
+        try:
+            self.get_db.write(
+                {'points': [point]},
+                {
+                    'db': kwargs.get('database') or self.db_name,
+                    'rp': kwargs.get('retention_policy'),
+                },
+            )
+        except InfluxDBServerError:
```

Comment:
why catch only `InfluxDBServerError`? 

I tried stopping the server on purpose and the task just failed without retrying.

I think here we should catch just `Exception`.

Reply:
I thought catching every exception might not be a good practice since if there is some other issue besides server load (say the `dict` we provide to the `write` method is of **invalid** format), it will stay hidden :no_mouth:. `InfluxDBClientError` was being raised in the case I mentioned but I agree if server goes down then the task won't be resilient which will defeat the purpose.

I am a bit unsure here, tbh as to what might be better for the long term. Currently, it will catch **every Exception**.